package.json

```diff
@@ -9,9 +9,9 @@
     "audit": "npm audit & npm run exec -- npm audit",
     "audit:fix": "npm audit fix & npm run exec -- npm audit fix",
     "build": "lerna run build",
-    "clean-install-packages": "npm run exec -- npm ci & npm ci",
+    "clean-install-packages": "npm run exec -- npm ci",
```

Comment:
But this won't run lerna bootstrap after. Why are you removing the top-level `npm ci`?

Reply:
This was accidental (good catch).  I did this because running npm install at the root and running it in the package directories does conflicting things.  Running at the root seems to remove some of the package dependencies and if you go into the package directory and run install after running at the root it will always find things that needs to be installed,  I meant to keep this as a local change until I had a chance to figure out what was going on and make a separate change.

Reply:
Lerna at least allows you to do normal npm commands (install, etc.) in the package directory (unlike Rush). You just need to do "lerna bootstrap" at the root afterwards to symlink the packages. I made this a post-install script, so it happens automatically after npm install at the root.